app/views/works/index.html.erb

```diff
@@ -0,0 +1,87 @@
+<% if flash[:success] %>
+<div class="alert alert-info">
+    <%= flash[:success] %>
+</div>
+<% end %>
+
+<% if flash[:error] %>
+<div class="alert alert-danger">
+    <%= flash[:error] %>
+</div>
+<% end %>
+
+<table class="table ">
+    <thead>
+        <tr>
+            <th>Votes</th>
+            <th>Title</th>
+            <th>Created By</th>
+            <th>Published</th>
+            <th>Upvote</th>
+
+        </tr>
+    </thead>
+    <tbody>
+        <h3> Top Albums </h3>
+        <% Work.albums_list.each do |work| %>
```

Comment:
As above, you essentially have the same code repeated 3 times. Consider how you may dry up this code, including modifying your model method to take the category as an argument.